web/pages/mensa.vue

```diff
@@ -6,7 +6,7 @@
     class="container-padding"
   >
     <h2>Mensa Wolfenbüttel</h2>
-    <span>Montag bis Freitag 11:15 - 14:15 Uhr</span>
+    <span><a href="https://www.stw-on.de/wolfenbuettel/essen/mensa/#c2368">Offizielle Mensa-Seite mit Öffnungszeiten</a></span>
```

Comment:
Das hier hab ich geändert, da die Zeiten variabel sind und gerade jetzt in der Corona-Zeit nicht gepasst haben. Wäre fast zur geschlossenen Mensa gegangen :D